app/views/static_pages/disclaimer.html.haml

```diff
@@ -1,56 +1,89 @@
-%p We respecteren de privacy van bezoekers van de website en dragen er zorg voor dat de persoonlijke |
-    informatie die u ons verschaft vertrouwelijk wordt behandeld. Verwerking van de persoonsgegevens |
-    gebeurt op een wijze, die in overeenstemming is met de eisen die de Wet Bescherming |
-    Persoonsgegevens stelt. Door gebruik te maken van de u-can-act website geeft u toestemming voor |
-    het plaatsen en aflezen van functionele en analytische cookies die helpen de kwaliteit en/of |
-    effectiviteit van de website te verbeteren. Een cookie is een eenvoudig klein bestandje dat met |
-    pagina’s van deze website wordt meegestuurd en door uw browser op de harde schijf van uw |
-    computer wordt opgeslagen. Via u-can-act worden analytische cookies geplaatst van het bedrijf |
-    Google. Deze cookies bevatten geen persoonsgegevens en zijn dus niet tot een individu te herleiden, |
-    omdat het IP-adres van iedere bezoeker is geanonimiseerd, en de gegevens in Google Analytics niet |
-    worden gedeeld. Deze cookies kunnen worden gebruikt voor statistische analyses van bezoek- en |
-    klikgedrag om de functionaliteit en effectiviteit van de website te kunnen optimaliseren. Deze |
-    cookies volgen u niet op andere websites. U kunt deze cookies uitzetten via uw browser (zie voor een |
-    toelichting van de consumentenbond #{link_to 'hier', 'https://www.consumentenbond.nl/internet-privacy/cookies-verwijderen/', target: '_blank'}. |
-    Wij behouden ons het recht voor deze privacy statement |
-    aan te passen. Wijzigingen zullen op deze website worden gepubliceerd. |
+%h2 Cookieverklaring
+%p Door gebruik te maken van de u-can-act website geeft u toestemming voor het |
+    plaatsen en aflezen van functionele en analytische cookies die helpen de |
+    kwaliteit en/of effectiviteit van de website te verbeteren. Een cookie is een |
+    eenvoudig klein bestandje dat met pagina’s van deze website wordt meegestuurd |
+    en door uw browser op de harde schijf van uw computer wordt opgeslagen. Via |
+    u-can-act worden analytische cookies geplaatst van het bedrijf Google. Deze |
+    cookies bevatten geen persoonsgegevens en zijn dus niet tot een individu te |
+    herleiden, omdat het IP-adres van iedere bezoeker is geanonimiseerd, en de |
+    gegevens in Google Analytics niet worden gedeeld. Deze cookies kunnen worden |
+    gebruikt voor statistische analyses van bezoek- en klikgedrag om de |
+    functionaliteit en effectiviteit van de website te kunnen optimaliseren. Deze |
+    cookies volgen u niet op andere websites. U kunt deze cookies uitzetten via uw |
+    browser (zie voor een toelichting van de consumentenbond #{link_to 'hier', 'https://www.consumentenbond.nl/internet-privacy/cookies-verwijderen/', rel: "noopener noreferrer", target: '_blank'}. |
+    Wij behouden ons het recht voor deze privacy statement aan te passen. |
+    Wijzigingen zullen op deze website worden gepubliceerd. |
```

Comment:
Dat van die druk- en zetfouten hieronder klinkt wel heel oud. Is daar niet iets hippers voor? Hoe kunnen we druk/zetfouten hebben op een website?